1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive = everyone.slice(0, 5); // complete this statement
+let lastFive = everyone.slice(2, 7); // complete this statement
```

Comment:
Nice. Try and avoid hardcoded/magic numbers.
Hint : Passing a negative index indicates an offset from the end of the array. A negative index of -3 means give me the last 3 elements of the array.

Reply:
Will try! Thank You!